cfgov/v1/models/base.py

```diff
@@ -0,0 +1,143 @@
+from django.db import models
+from django.http import Http404
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+
+from wagtail.wagtailcore.models import Page, PagePermissionTester, UserPagePermissionsProxy
+from wagtail.wagtailcore.url_routing import RouteResult
+from wagtail.wagtailadmin.edit_handlers import FieldPanel, MultiFieldPanel
+
+from taggit.models import TaggedItemBase
+from modelcluster.fields import ParentalKey
+from modelcluster.tags import ClusterTaggableManager
+
+
+class CFGOVAuthoredPages(TaggedItemBase):
+    content_object = ParentalKey('CFGOVPage')
+
+    class Meta:
+        verbose_name = _("Author")
+        verbose_name_plural = _("Authors")
+
+
+class CFGOVTaggedPages(TaggedItemBase):
+    content_object = ParentalKey('CFGOVPage')
+
+    class Meta:
+        verbose_name = _("Tag")
+        verbose_name_plural = _("Tags")
+
+
+class CFGOVPage(Page):
+    authors = ClusterTaggableManager(through=CFGOVAuthoredPages, blank=True,
+                                     verbose_name='Authors',
+                                     help_text='A comma separated list of authors.',
+                                     related_name='authored_pages')
+    tags = ClusterTaggableManager(through=CFGOVTaggedPages, blank=True,
+                                  related_name='tagged_pages')
+    shared = models.BooleanField(default=False)
+
+    # This is used solely for subclassing pages we want to make at the CFPB.
+    is_creatable = False
```

Comment:
oh i see what your on about. the `eventpage` is inheriting `is_creatable = True` from the wagtail `Page` model